pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The CDI Authors.
+Copyright The Kubernetes Authors.
```

Comment:
I am fairly certain this should remain the CDI authors, not the kubernetes authors.

Reply:
Yeah, I wondered how I could prevent the k8s code-generator to change it. Any idea how a custom copyright can be configured?

Reply:
ok, used 'make generate' instead (which using the 'custom-boilerplate.go.txt')